iml-agent/src/action_plugins/check_ha.rs

```diff
@@ -38,14 +48,31 @@ fn create<'a>(elem: &Element, group: impl Into<Option<&'a str>>) -> ResourceAgen
 
 fn process_resource_list(output: &[u8]) -> Result<Vec<ResourceAgentInfo>, ImlAgentError> {
     let element = Element::from_reader(output)?;
-
+    let templates: HashMap<String, ResourceAgentType> = element
+        .find_all("template")
+        .map(|elem| {
+            (
+                elem.get_attr("id").unwrap_or("").to_string(),
+                ResourceAgentType::new(
```

Comment:
Is it always safe to assume if we hit a `template` that it will alway have these values?